tests/manage/mcg/test_bucket_policy.py

```diff
@@ -311,8 +311,421 @@ def test_anonymous_read_only(self, mcg_obj, bucket_factory):
 
         # Admin writes an object to bucket
         logger.info(f'Writing object on bucket: {s3_bucket.name} by admin')
-        assert helpers.s3_put_object(mcg_obj, s3_bucket.name, object_key, data), "Failed: Put Object"
+        assert helpers.s3_put_object(mcg_obj, s3_bucket.name, object_key, data), "Failed: PutObject"
 
         # Reading the object by anonymous user
         logger.info(f'Getting object by user: {user.email_id} on bucket: {s3_bucket.name} ')
         assert helpers.s3_get_object(user, s3_bucket.name, object_key), f"Failed: Get Object by user {user.email_id}"
+
+    @pytest.mark.polarion_id("OCS-2140")
+    @tier1
+    def test_bucket_website_and_policies(self, mcg_obj, bucket_factory):
+        """
+        Tests bucket website bucket policy actions
+        """
+        website_config = {
+            'ErrorDocument': {'Key': 'error.html'},
+            'IndexDocument': {'Suffix': 'index.html'},
+        }
+        index = "<html><body><h1>My Static Website on S3</h1></body></html>"
+        error = "<html><body><h1>Oh. Something bad happened!</h1></body></html>"
```

Comment:
Would it make sense to save these in defaults.py?

Reply:
yup can be moved to defaults